scripts/mumbai/deploy.ts

```diff
@@ -12,11 +12,15 @@ async function main() {
   const consumer = await OracleConsumerContract.deploy(attestor);
   await consumer.deployed();
   const finalMessage = dedent`
-    🎉 Your Consumer Contract has been deployed, check it out here: https://mumbai.polygonscan.com/address/${consumer.address}
-    
-    You also need to set up the consumer contract address in your .env file:
-    
-    MUMBAI_CONSUMER_CONTRACT_ADDRESS=${consumer.address}
+    🎉 Your Consumer Contract has been deployed successfully 🎉
+
+    address ${consumer.address}
+
+    Check it out here: https://mumbai.polygonscan.com/address/${consumer.address}
+
+    You can continuing deploy the default Phat Contract with following command:
+
+    npx @phala/fn upload -b --mode dev --consumerAddress=${consumer.address}
```

Comment:
add default secrets.
```suggestion
    npx @phala/fn upload -b --mode dev --consumerAddress=${consumer.address} --coreSettings='{"apiUrl": "https://api.airstack.xyz/gql", "apiKey": "3a41775a358a4cb99ca9a29c1f6fc486"}'
```